test/parallel/test-buffer-readintle.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+require('../common');
+const assert = require('assert');
+
+// testing basic functionality of readIntLE()
+
+const buf = Buffer.from([42, 84, 167, 127]);
+const result = buf.readIntLE(0);
```

Comment:
`readIntLE()` requires the second parameter, which is `bytelength`. This could be `buf.readIntLE(0, 1)`.